packages/shared/src/components/RenderMarkdown.tsx

```diff
@@ -137,23 +141,46 @@ const RenderMarkdown = ({
             loadLanguages();
           }
 
-          return !inline && language ? (
-            <SyntaxHighlighterAsync
-              {...props}
-              customStyle={containerReset}
-              language={language}
-              useInlineStyles={false}
-              wrapLongLines
-              codeTagProps={{
-                className: codeClassName,
-              }}
-            >
-              {String(children).replace(replaceNewLineRegex, '')}
-            </SyntaxHighlighterAsync>
-          ) : (
-            <code {...props} className={codeClassName}>
-              {children}
-            </code>
+          return (
+            <>
+              {language && (
```

Comment:
I think we should still render the option to copy even if we don't know the language

Reply:
👍